pkg/state/impl/inmem/collection.go

```diff
@@ -456,7 +456,13 @@ func (collection *ResourceCollection) WatchAll(ctx context.Context, singleCh cha
 					}
 				}
 
-				if !channel.SendWithContext(ctx, singleCh, state.Event{Type: state.Bootstrapped}) {
+				if !channel.SendWithContext(
+					ctx, singleCh,
+					state.Event{
+						Type:     state.Bootstrapped,
+						Resource: resource.NewTombstone(resource.NewMetadata(collection.ns, collection.typ, "", resource.VersionUndefined)),
```

Comment:
Why send a tombstone here?

Reply:
Nvm, saw the state-etcd PR which explains it.

Reply:
well, the idea is to send which Watch bootstrapped, as if you Watch multiple resource types, and send all watch events to the same channel, there is no way to tell which one boostrapped.

Tombstone is used just as something which carries the minimum Resource metadata (we need to send namespace & type only).